src/libraries/Common/tests/Tests/System/Net/Http2/ReadMe.SharedCode.md

```diff
@@ -1,3 +1,5 @@
-The code in this directory is shared between CoreFx and AspNetCore. This contains tests for HTTP/2 protocol infrastructure such as HPACK. Any changes to this dir need to be checked into both repositories.
+The code in this directory is shared between the runtime libraries and AspNetCore. This contains tests for HTTP/2 protocol infrastructure such as HPACK. Any changes to this dir need to be checked into both repositories.
 
-See corefx\src\Common\src\System\Net\Http\Http2\ReadMe.SharedCode.md or AspNetCore\src\Shared\Http2\ReadMe.SharedCode.md for additional details.
\ No newline at end of file
+For additional details see:
+- https://github.com/dotnet/runtime/blob/master/src/libraries/Common/src/System/Net/Http/Http2/ReadMe.SharedCode.md
+- https://github.com/aspnet/AspNetCore/blob/master/src/Shared/Http2/ReadMe.SharedCode.m
```

Comment:
Can you also update the AspNetCore version of this file as discussed in this readme?

Reply:
These were intentionally repo relative file paths rather than branch relative urls. Linking to a specific branch means the link gets out of sync with the source.

Reply:
I can update the AspNetCore file directly on Github.  Also I just noticed the second link is missing the letter d at the very end.

Reply:
I just grokked your meaning with regards to repo vs branch.  It's fixed, Thanks!